tests/components/fitbit/test_sensor.py

```diff
@@ -244,11 +244,27 @@ async def test_device_battery_level(
 
 
 @pytest.mark.parametrize(
-    ("monitored_resources", "profile_locale", "expected_unit"),
+    (
+        "monitored_resources",
+        "profile_locale",
+        "configured_unit_system",
+        "expected_unit",
+    ),
     [
-        (["body/weight"], "en_US", "kg"),
-        (["body/weight"], "en_GB", "st"),
-        (["body/weight"], "es_ES", "kg"),
+        # Defaults to home assistant unit system unless UK
```

Comment:
I'm considering adding another test that chagnes the hass metrics to imperial...